docs/artifacts/upgrade-protocol.html

```
@@ -0,0 +1,229 @@
+<title>The Upgrade Protocol</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)